diff -urN glademm-0.6.2b.org/configure.in glademm-0.6.2b/configure.in
--- glademm-0.6.2b.org/configure.in	Sun Feb 17 20:19:07 2002
+++ glademm-0.6.2b/configure.in	Sun Feb 17 20:20:48 2002
@@ -185,7 +186,7 @@
     AC_MSG_RESULT(gnome-config not found)
 elif test -r "$gnomelib/gnomemmConf.sh" 
 then
-    source $gnomelib/gnomemmConf.sh
+    . $gnomelib/gnomemmConf.sh
     if test "x$MODULE_VERSION" = "x"
     then
	AC_MSG_RESULT(no version info available)
